code-challenges.test.js

```diff
@@ -12,56 +14,166 @@
-  thursday: 30,
-  friday: 15,
-  saturday: 60
-}
-// Expected output: [15, 15, 20, 30, 30, 60, 90]
-
-const studyMinutesWeek2 = {
-  sunday: 100,
-  monday: 10,
-  tuesday: 45,
-  wednesday: 60,
-  thursday: 20,
-  friday: 15,
-  saturday: 65
-}
-// Expected output: [10, 15, 20, 45, 60, 65, 100]
+describe ("numArr", () => {
+  it ("takes the values from the key value pairs and returns them in an array, sorted numerically", () => {
+    const studyMinutesWeek1 = {
+      sunday: 90,
+      monday: 30,
+      tuesday: 20,
+      wednesday: 15,
+      thursday: 30,
+      friday: 15,
+      saturday: 60
+    }
+    // Expected output: [15, 15, 20, 30, 30, 60, 90]
+    const studyMinutesWeek2 = {
+      sunday: 100,
+      monday: 10,
+      tuesday: 45,
+      wednesday: 60,
+      thursday: 20,
+      friday: 15,
+      saturday: 65
+    }
+    // Expected output: [10, 15, 20, 45, 60, 65, 100]
+    expect(numArr(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
+    expect(numArr(studyMinutesWeek2)).toEqual([10, 15, 20, 45, 60, 65, 100])
+  })
+
+})
+
+// FAIL  ./code-challenges.test.js
+// fibArr
+//   ✓ Takes in the length of the array and returns an array with every index as the sum of the previous two (5 ms)
+// numArr
+//   ✕ takes the values from the key value pairs and returns them in an array, sorted numerically (3 ms)
+//   ReferenceError: numArr is not defined
+
 
 // b) Create the function that makes the test pass.
+const numArr = (object) => {
+  return Object.values(object).sort((a, b) => a - b)
+}
 
-// --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+// PASS  ./code-challenges.test.js
+// fibArr
+//   ✓ Takes in the length of the array and returns an array with every index as the sum of the previous two (3 ms)
+// numArr
+//   ✓ takes the values from the key value pairs and returns them in an array, sorted numerically (2 ms)
 
-// a) Create a test with expect statements for each of the variables provided.
 
-const accountTransactions1 = [100, -17, -23, -9]
-// Expected output: [100, 83, 60, 51]
 
-const accountTransactions2 = [250, -89, 100, -96]
-// Expected output: [250, 161, 261, 165]
 
-const accountTransactions3 = []
-// Expected output: []
+// --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+
+// PseudoCode:
+  // Input: the input will be an array of numbers
+  // Output: an array of the sums of the numbers in the input array
+  // Process: use .map to iterate through the array, and add the values from all of the previous indexes to the current index as the values in the new array. first, set conditional statement to account for an empty array. Then, declare a variable as the starting position, and use the addition assignment operator to add the values from the previosu indexes to the current one. 
```

Comment:
The explanation does not completely match what your function is doing, specifically how you are keeping track of the accumulating sum